### PR TITLE
TEQ-4 project setup instructions

### PR DESCRIPTION
**Pull request by jbradberry**

### CODE REVIEW

File: docs/project_setup.rst [hunk 1]
```diff
@@ -101,8 +101,7 @@ Adding Tequila to a Django Project
    catch-all playbook that uses the include directive to pull in at
    least the common, db, web, worker, and queue playbooks).  Other
    playbooks should be created as needed to fill other
-   project-specific needs.  **TODO:** Create this playbooks/ directory
-   and populate.
+   project-specific needs.
```
Comment (dchukhin) [inline docs/project_setup.rst]: Would it make sense to mention the directory of typical playbooks here, or perhaps somewhere else?

Comment (jbradberry) [reply docs/project_setup.rst]: Line 94.

Comment (dchukhin) [reply docs/project_setup.rst]: 👀

**Review by mlavin — APPROVED** [review]
Looks good to me :books: :ship:

**Pull request merged**